### ISSUE TITLE
Update to maintained version - any quick help?

### ISSUE THREAD

**Issue by polarche** [DoctorD1501/JAVMovieScraper#344]
I have been this program for a few years.  However I believe there were changes to various websites which made the v.0.3.7 having some problems to some common websites like dmm.  I read the posts and realised there were changes needed to make the program able to run again.
Unfortunately I am not familiar with programming, would any expert have any quick tips or any usable jar for sharing?  Much appreciated help...thanks

**Comment by mesoller** [comment]
Yes, there seems like an update to dependencies 6 months back but not new release have been compile as jar. I also dont know how to compile it

**Comment by RabidWolf** [comment]
https://github.com/DoctorD1501/JAVMovieScraper/issues/335

Scroll down and zuko7177 has put up a jar with the pull requests added along with install instructions.
